### PR TITLE
[hack] Organisation stuffs

### PR DESCRIPTION
**Pull request by clicktravel-martindimitrov**

### CODE REVIEW

File: cheddar/cheddar-metrics-intercom/src/main/java/com/clicktravel/cheddar/metrics/intercom/IntercomMetricCollector.java [hunk 1]
```diff
@@ -129,4 +127,23 @@ private void createOrUpdateIntercomUser(final MetricUser user) {
...
+        final String companyID = organisation.id;
+        final String name = organisation.name;
+
+        company.setCompanyID(companyID);
+        company.setName(name);
+        try {
+
+            Company.create(company);
+        } catch (final Exception e) {
+            logger.warn("Error creating/updating a Intercom company - " + company + " - " + e.getLocalizedMessage());
```
Comment (clicktravel-basharat) [inline cheddar/cheddar-metrics-intercom/src/main/java/com/clicktravel/cheddar/metrics/intercom/IntercomMetricCollector.java]: debug not warn.

File: cheddar/cheddar-metrics-intercom/src/main/java/com/clicktravel/cheddar/metrics/intercom/IntercomMetricCollector.java [hunk 2]
```diff
@@ -129,4 +127,23 @@ private void createOrUpdateIntercomUser(final MetricUser user) {
...
+        }
+        final Company company = new Company();
+
+        final String companyID = organisation.id;
+        final String name = organisation.name;
+
+        company.setCompanyID(companyID);
+        company.setName(name);
+        try {
+
```
Comment (clicktravel-basharat) [inline cheddar/cheddar-metrics-intercom/src/main/java/com/clicktravel/cheddar/metrics/intercom/IntercomMetricCollector.java]: get rid of this new line

**Comment by clicktravel-martindimitrov** [comment]
@clicktravel-basharat 
Done.

File: cheddar/cheddar-metrics-intercom/src/main/java/com/clicktravel/cheddar/metrics/intercom/IntercomMetricCollector.java [hunk 3]
```diff
@@ -126,7 +124,25 @@ private void createOrUpdateIntercomUser(final MetricUser user) {
...
+        }
+    }
+
+    private void createOrUpdateIntercomCompany(final MetricOrganisation organisation) {
+        if (organisation == null) {
+            return;
+        }
+        final Company company = new Company();
+
+        final String companyID = organisation.id;
```
Comment (clicktravel-basharat) [inline cheddar/cheddar-metrics-intercom/src/main/java/com/clicktravel/cheddar/metrics/intercom/IntercomMetricCollector.java]: companyId

File: cheddar/cheddar-metrics-intercom/src/main/java/com/clicktravel/cheddar/metrics/intercom/IntercomMetricCollector.java [hunk 4]
```diff
@@ -126,7 +124,25 @@ private void createOrUpdateIntercomUser(final MetricUser user) {
...
+        final String name = organisation.name;
+
+        company.setCompanyID(companyID);
+        company.setName(name);
+        try {
+            Company.create(company);
+        } catch (final Exception e) {
+            logger.debug("Error creating/updating a Intercom company - " + company + " - " + e.getLocalizedMessage());
```
Comment (clicktravel-basharat) [inline cheddar/cheddar-metrics-intercom/src/main/java/com/clicktravel/cheddar/metrics/intercom/IntercomMetricCollector.java]: getLocalizedMessage - > getMessage

**Comment by clicktravel-basharat** [comment]
Couple of comments.

**Pull request merged**